costmap:
  costmap:
    ros__parameters:
      update_frequency: 5.0
      publish_frequency: 1.0
      global_frame: map
      robot_base_frame: map
      use_sim_time: False
      robot_radius: 0.22
      resolution: 0.05
      track_unknown_space: true
      plugins: ["static_layer", "workcell_cost_layer", "inflation_layer"]
      static_layer:
        plugin: "nav2_costmap_2d::StaticLayer"
        map_subscribe_transient_local: True
      inflation_layer:
        plugin: "nav2_costmap_2d::InflationLayer"
        cost_scaling_factor: 3.0
        inflation_radius: 0.55
      workcell_cost_layer:
        plugin: "nav2_costmap_2d::WorkcellCostLayer"
        enabled: True
        shape: 1 #shape 0 = tunnel, 1 = square, 2 = circle
        cost: 254 #LETHAL_COST = 254
      always_send_full_costmap: True
  global_costmap_client:
    ros__parameters:
      use_sim_time: False
  global_costmap_rclcpp_node:
    ros__parameters:
      use_sim_time: False

global_map_server:
  ros__parameters:
    use_sim_time: False
    yaml_filename: "src/spice_nav/maps/C4.yaml"
    frame_id: map
    topic_name: map